uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaJMXNavigatorTest.java

```diff
@@ -0,0 +1,139 @@
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.cleanworkspace.CleanWorkspaceRequirement.CleanWorkspace;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+import org.jboss.tools.fuse.reddeer.LogGrapper;
+import org.jboss.tools.fuse.reddeer.condition.JMXConnectionIsAvailable;
+import org.jboss.tools.fuse.reddeer.requirement.CamelExampleRequirement;
+import org.jboss.tools.fuse.reddeer.requirement.CamelExampleRequirementMatcher;
+import org.jboss.tools.fuse.reddeer.requirement.JolokiaConfiguration;
+import org.jboss.tools.fuse.reddeer.requirement.CamelExampleRequirement.CamelExample;
+import org.jboss.tools.fuse.reddeer.view.FuseJMXNavigator;
+import org.jboss.tools.fuse.reddeer.wizard.JMXNewConnectionWizard;
+import org.jboss.tools.fuse.reddeer.wizard.JMXNewConnectionWizard.ConnectionType;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * This test requires an external Camel route specified by the requirement
+ * restriction. The Camel route is started and stopped by the requirement logic.
+ * 
+ * @author djelinek
+ *
+ */
+@CleanWorkspace
+@CamelExample(useJolokia = true)
+@RunWith(RedDeerSuite.class)
+public class JolokiaJMXNavigatorTest {
+
+	private static Logger log = Logger.getLogger(JMXNavigatorTest.class);
+
+	@InjectRequirement
+	private CamelExampleRequirement camelExample;
+
+	@RequirementRestriction
+	public static RequirementMatcher getRequirementMatcher() {
+		return new CamelExampleRequirementMatcher("camel-example-spring-boot", "2.20.1");
+	}
+	
+	/**
+	 * Prepares test environment
+	 */
+	@Before
+	public void defaultSetup() {
+		log.info("Deleting Error Log.");
+		LogView errorLog = new LogView();
+		errorLog.open();
+		errorLog.deleteLog();
+	}
+
+	/**
+	 * <p>
+	 * Test tries to create new <i>Jolokia connection</i> in JMX Navigator and access route nodes.
+	 * </p>
+	 * <ul>
+	 * <li>Create a new connection in JMX Navigator</li>
+	 * <li>New connection... --> <i>Jolokia Connection</i></li>
+	 * <li>Provide the name you want</li>
+	 * <li>Use Jolokia URL from Console View (by default <i>http://127.0.0.1:8778/jolokia</i>)</li>
+	 * <li>Select <i>Do NOT verify SSL Certificates</i></li>
+	 * <li>Click Finish</li>
+	 * <li>Expand User-Defined Connections --> double-click on the connection to activate it</li>
+	 * <li>Try to work with it (accessing nodes, perform available operations, ...)</li>
+	 * </ul>
+	 * 
+	 * @throws Exception
+	 */
+	@Test
+	public void testJolokiaAcccessingCamelRoute() throws Exception {
+
+		JolokiaConfiguration conf = camelExample.getConfiguration().getJolokiaConfiguration();
+
+		log.info("Try to open/activate 'JMX Navigator' view");
+		FuseJMXNavigator jmxView = new FuseJMXNavigator();
+		jmxView.activate();
+		jmxView.clickNewConnection();
+
+		log.info("Try to create a new jolokia connection");
+		JMXNewConnectionWizard connectionWizard = new JMXNewConnectionWizard("Create JMX Connection");
```

Comment:
The wizard could have different name than _Create JMX Connection_? What about move the name into constructor `super("Create JMX Connection");` and the wait condition from next line too?